src/styles/main.scss

```
@@ -1,3 +1,203 @@
-body {
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
+
+html,
+body {
+  font-family: "Roboto", Helvetica, sans-serif;
+}
+
+.container {
+  display: grid;
+  padding: 50px 40px;
+  grid-template-columns: repeat(4, 200px);
+  row-gap: 46px;
+  column-gap: 48px;
+  justify-content: center;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.logo {
+  display: flex;
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+}
+
+.nav__item {
+  height: 60px;
+  display: flex;
+  align-items: center;
+
+  &:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &:last-child {
+    margin-right: 0;
+  }
+}
+
+.nav__link {
+  height: 60px;
```

```suggestion
  padding: 23px 0;
```
set directly height is bad practice. instead make vertical paddings